### ISSUE TITLE
Create Bone enum.

### ISSUE THREAD

**Issue by ikkentim** [ikkentim/SampSharp#75]
see http://wiki.sa-mp.com/wiki/Bone_IDs

**Comment by rrev** [comment]
There is already some come which uses `BoneId` shall we maintain backward-compatibility even if we are still in 0.x versions? Or we can just ignore it for now to allows us to release a very clean framework code for 1.0 etc.

Right now I have tried to keep retro-compatibility by providing an overload which accepts `int` or `Bone` and inside PlayerEditAttachedObjectEventArgs I've marked `BoneId` as `Obsolete`.

**Personally** I would break retro-compatibility for now since we are still in 0.x versions and it wouldn't break a lot of code anyway

Check this compare page https://github.com/rrev/SampSharp/compare/bone

Maybe i'm worrying for nothing
